CONTRIBUTING.md

```diff
@@ -32,16 +32,17 @@ Please keep the following in mind:
 
 - Please run the following commands before submitting your PR:
     * `pub run test` - All the tests for Dart exercises can be run from the top level of the repo with this command
-    * `bin/configlet lint .` - Checks the config.json for formatting issues
+    * `bin/fetch-configlet` - Downloads the latest version of configlet and stores it in the bin/ directory
+    * `bin/configlet lint --track-id dart .` - Checks the config.json for formatting issues
     * `bin/configlet fmt .` - Formats the config.json
     * `bin/check_formatting` - Checks All the Dart files for formatting issues
```

Comment:
check_formatting is not an executable anymore.

Reply:
It hasn't been merged in, so currently in master, it is. Is it possible that PR will be merged in before this one?

Reply:
Well, it depends on us really. It's almost complete though. Plus IMO making changes to CONTRIBUTING.md should be its own PR. Since we already have one why not wait? For example if we create a post-installation script in the future should the INSTALLATION.md be updated in the same PR or different? What if 2 different contributors are contributing 2 different post-installation scripts both completely necessary and at the same time. It really comes down to us which script should be documented when, where and how.

Reply:
For me, if a script is added or changed, the related documentation should be changed. If we have two scripts for post-installation that do different tasks. And for some reason it was valid for the scripts to be separate and not one file, then each one will need to update documentation. Then whichever one went last would have to incorporate the changes from the other.

It's just simpler to do that in the same PR, unless the work for documentation was significant in and of itself. Like multiple pages being affected.

We can certainly delay this ticket, but I'm going to ask @raphalvessa to commit the related changes.

Reply:
Whoops, I only just saw my mention. Is this related to my changes in the presubmit scripts? I actually edited this file in my PR to use the new scripts. Feel free to merge this when it's ready, I can later resolve any possible merge conflicts in my PR.